lib/Client.js

```diff
@@ -2071,6 +2076,36 @@ class Client extends EventEmitter {
         }));
     }
 
+    _formatAllowedMentions(allowed) {
+        if(allowed) {
+            const result = {
+                parse: []
+            };
+            if(allowed.everyone) {
+                result.parse.push("everyone");
+            }
+            if(allowed.roles === true) {
+                result.parse.push("roles");
+            } else if(Array.isArray(allowed.roles)) {
+                if(allowed.roles.length > 100) {
+                    throw new Error("Allowed role mentions cannot exceed 100.");
+                }
+                result.roles = allowed.roles;
+            }
+            if(allowed.users === true) {
+                result.parse.push("users");
+            } else if(Array.isArray(allowed.users)) {
+                if(allowed.roles.length > 100) {
+                    throw new Error("Allowed user mentions cannot exceed 100.");
+                }
+                result.users = allowed.users;
+            }
+            return result;
+        } else if(this.options.allowedMentions) {
+            return this.options.allowedMentions;
```

Comment:
I think this might break if im not mistaken since you default `roles: true` and `users: `true` instead of `parse: ['roles', 'users']`

Reply:
You're mistaken, when they're `true` they're added to the `parse` array.